compose.yaml

```diff
@@ -34,4 +34,4 @@ services:
 
 volumes:
     pgsql-data: ~
-    rabbitmq-data: ~
+    rabbitmq-data: ~    
```

Comment:
I know it's a small thing, just a tab. But if you're not editing anything here, there shouldn't be a space. You should always check that you're only committing the files you're changing.